frontend/src/firebase/index.ts

```diff
@@ -10,6 +10,8 @@ if (process.env.NODE_ENV === 'production' && process.env.REACT_APP_IS_STAGING !=
     projectId: 'dti-samwise',
     storageBucket: 'dti-samwise.appspot.com',
     messagingSenderId: '114434220691',
+    appId: '1:114434220691:web:01579d764b20fb0aeb6366',
```

Comment:
Just double checking that you intended to expose these.

Reply:
Yes, this is specified in the analytics docs: https://firebase.google.com/docs/analytics/get-started